app/views/variable_lookup_keys/welcome.html.erb

```diff
@@ -0,0 +1,11 @@
+<% content_for(:title, _("Smart Variables")) %>
+<div class="blank-slate-pf">
+  <div class="blank-slate-pf-icon">
+    <%= icon_text("info-circle", "", :kind => "fa") %>
+  </div>
+  <h1><%= _('Smart Variables') %></h1>
+  <p><%= _("Smart variables are a tool to provide global parameters (key/value data), normally to your Puppet ENC,
```

Comment:
smart variables are usually not global, they use the same rules as smart class parameters, the main difference is that they end up in different place in ENC so they end up in the same namespace as Parameters (global params, host params etc)

Reply:
They are global from a Puppet perspective - that's what the Parameters block is. For Foreman they are *both* global *and* assigned to a class - If you try to have the same name in two smart vars, it'll clash even if they're assigned to different classes (I've no idea if that actually creates an error, but it won't work in the ENC for sure).

Getting all that nuance into a short text line is hard though....

Reply:
Not bad. Maybe change `to a global namespace of a puppet run through` to `to the top-level (global) scope of your Puppet ENC`?

Reply:
👍